registry/registry.json

```diff
@@ -88,12 +92,17 @@
       "type": "registry:block",
       "title": "MCP Starter Runtime",
       "description": "A runtime for CopilotKit with MCP.",
-      "dependencies": ["@copilotkit/runtime"],
+      "dependencies": ["@copilotkit/runtime", "@modelcontextprotocol/sdk"],
       "files": [
         {
           "path": "registry/quickstarts/mcp-starter/runtime.ts",
           "type": "registry:lib",
           "target": "app/api/copilotkit/route.ts"
+        },
+        {
+          "path": "registry/quickstarts/mcp-starter/utils/mcp-client.ts",
+          "type": "registry:lib",
+          "target": "app/utils/mcp-client.ts"
```

Comment:
You shouldn't need a target for this, I'd just let ShadCN take care of the routing